### ISSUE TITLE
[Feature] Support min_lr clipping in lr_scheduler (MMDet3D related problem)

### ISSUE THREAD

**Issue by Wuziyi616** [open-mmlab/mmcv#994]
**Hi, I am Ziyi, a developer of MMDet3D. Can we support learning rate clipping in lr_scheduler? For example, when I use StepLR which decays by 0.7 every 10 epochs with a `init_lr==1e-3`, I would like to keep my `lr==1e-5` after it decays to some values lower than `1e-5`. Such min_lr clipping is common in several 3D point cloud papers.**

**Motivation**
For example, in the PointNet++ official code, they decay lr by 0.7 every 10 epochs and clip a minium lr of 1e-5 as shown [here](https://github.com/charlesq34/pointnet2/blob/master/scannet/train.py#L85). This is widely adopted in other point cloud related papers like PointNet (see [here](https://github.com/charlesq34/pointnet/blob/master/train.py#L80)), DGCNN, etc.

**Related resources**
I think it's quite easy to implement. We can add a `min_lr` args to `LrUpdaterHook`, and use a `max(lr, self.min_lr)` to determine the return value in `get_lr`? However, I am not sure whether it will cause other questions because I am not very familiar with MMCV.

**Additional context**
Is there any existing way to achieve my feature request? If you not yet and you think my solution is okay, I can create a PR to support it. Thanks!

**Comment by zhouzaida** [comment]
hi, @Wuziyi616 , maybe you can create a pr to solve it.

**Comment by Wuziyi616** [comment]
@zhouzaida Thanks for your reply! I have created a PR, please review it. Thanks!
